watt.py

```diff
@@ -172,6 +173,27 @@ def download_gcs_blob_to_tmp_file(self, blob_path: str, tmp_file_name: str) -> N
         blob = bucket.blob(blob_str)
         blob.download_to_filename(tmp_file_name)
 
+    @staticmethod
+    def read_in_chunks(file_object, chunk_size=1024*1024):
+        while True:
+            data = file_object.readlines(chunk_size)
+            if not data:
+                break
+            yield data
+
+    @staticmethod
+    def compare_files(file1: io.TextIOWrapper, file2: io.TextIOWrapper, line_skip_regex_str: str = None) -> int:
+        line_skip_regex = None
```

Comment:
Why not something like:
```
line_skip_regex = re.compile(line_skip_regex_str) if line_skip_regex_str is not None else None
```
I feel like I've had issues sometimes with this pattern dealing with `None` objects but does it work?